app/2.0/docs/devguide/settings.md

```diff
@@ -4,9 +4,45 @@ title: Global Polymer settings
 
 Document-level global Polymer settings can be set
 by creating a `Polymer` object on window before importing the Polymer
-library:
+library, or by calling a setter.
 
+
```

Comment:
@azakus can you review this file? Trying to add new instructions/samples for calling the setter.

Reply:
LGTM